src/main/java/org/eclipse/microprofile/starter/rest/APIEndpointV6.java

```diff
@@ -19,30 +19,12 @@
  */
 package org.eclipse.microprofile.starter.rest;
 
-import javax.inject.Inject;
```

Comment:
Why did you delete the block? It is literally a blank java file.

Reply:
I see you make this file as the latest.

Reply:
@Emily-Jiang The bug was a result of a misunderstanding. The way it is setup now ( and I am not saying that it is the best way to do it, but it is what we have) is that each APIEndpointV* lists *differences* in methods compared to the latest.
Hence Version 6 is empty now as it has no differences.
When someone goes and touches the current API in latest, he/she should also edit the V6 file and add the difference there.
Then a new V7 file will be empty and  /api/ will point to to the same as /api/7/....